CMakeLists.txt

```diff
@@ -155,6 +155,11 @@ if(NOT OPTION_DEDICATED)
             find_package(ICU OPTIONAL_COMPONENTS i18n)
         endif()
     endif()
+    find_package(Ogg)
+    find_package(Opus)
+    if(Opus_FOUND)
+        find_package(OpusFile)
+    endif()
```

Comment:
For me, OpusFile should be the only responsible to search for Ogg and Opus, and it should include the deps in OpusFile::opusfile target.